week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,9 +49,16 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
-}
-
+  persentageArray = [];
+  for(i = 0; i < arr.lenght; i++) {
+     checkArray = arr[i] > 100;
+     checkDecimal = arr[i] % 1 == 0;
+     if(checkArray || checkDecimal) {
+       persentageArray.push(arr[i].toFixed(2))
+     }
+    }
+    return persentageArray;
+  }
```

Comment:
This looks good but I'd pay attention to a few things:

- Don't forget to use `let` or `const`, otherwise the variables you use will be global, which is not a good idea.
- `arr.lenght` has a typo and will evaluate to `undefined`
- I don't think "numbers greater 100 must be replaced with 100" is implemented correctly
- What's the purpose of `checkDecimal`?
- Make sure to indent your code to make it more readable